### PR TITLE
IPFS Swag package

### LINKED ISSUE #300: IPFS Swag package.
## Wild swag opportunity appeared! :o

- [✔️] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : IPFS

Q. Where have you seen the announcement (URL)?  
A. : https://github.com/ipfs/community/#swag

Q. What kind of swag (clothing, stickers, ...)?  
A. : I´ve received a T-shirt with the IPFS logo, and 50 IPFS logo stickers.

Q. Have you received swag from this?  
A. : Yes.

Q. How long did it take?  
A. : Around less than one week.

Q. Does it ship worldwide?  
A. : Yes. It looks like it. I´m from France for reference.

I don’t even feel that confortable to post this opportunity as free T-shirts seems expensive for me, but hey, It's merchandising, and its public!
Also, More swag could potentially be obtained, as no precise description is done in the post. So, I was quite a bit hesitant as I was just expecting to receive only some stickers.

- [✔️] I've checked that the promotion is legit.
- [✔️] I've checked that it's still available to the public.

You´re welcome, template people.

### PR DESCRIPTION
**Pull request by PoorPocketsMcNewHold**
- [✔️] I've checked that this isn't a new swag opportunity proposal.
- [✔️] I've checked that this isn't a duplicate pull request.

Fixes #300 
The reason why i also got the Image change in this pull request is like you can see, due to a automated pull request of the ImgBotApp bot. If needed, I can just remove it, since it isn´t part of the main pull request being this new entry.

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -313,5 +313,14 @@
     "image": "https://developers.whatismybrowser.com/static/main/content/api/swag/whatismybrowser-stickers.jpg",
     "dateAdded": "2018-10-12T13:05:05.000Z",
     "tags": ["stickers"]
+  },
+  {
+    "name": "IPFS",
```
Comment (aslafy-z) [inline data.json]: Please move that entry to keep the list ordered alphabetically.

File: data.json [hunk 2]
```diff
@@ -313,5 +313,14 @@
...
+    "difficulty": "easy",
+    "description": "Contact the IPFS team via e-mail, and ask them kindly for some swag!",
+    "reference": "https://github.com/ipfs/community/#swag",
+    "image": "https://user-images.githubusercontent.com/13170204/63634719-9a530880-c65a-11e9-9937-05a70564d1e8.jpg",
```
Comment (aslafy-z) [inline data.json]: Let's use an url that we are sure of.

```suggestion
    "image": "https://i.imgur.com/sGUHD1P.jpg",
```

File: data.json [hunk 3]
```diff
@@ -313,5 +313,14 @@
...
+    "dateAdded": "2019-09-30T16:16:05.000Z",
```
Comment (vikaspotluri123) [inline data.json]: @aslafy-z As far as I know we're using the issue open date, can you confirm? Thanks!

Comment (aslafy-z) [reply data.json]: Yes, we are

Comment (aslafy-z) [inline data.json]:
```suggestion
    "dateAdded": "2019-08-20T19:20:42.000Z",
```

Comment (PoorPocketsMcNewHold) [reply data.json]: I´ve committed the following suggested @aslafy-z changes on the date. To be honest, I was pretty confuse on the format used for it.

File: data.json (hunk 2 above)
Comment (PoorPocketsMcNewHold) [reply data.json]: Done.

File: data.json (hunk 1 above)
Comment (PoorPocketsMcNewHold) [reply data.json]: I´ve placed it, right after the H. I hope i still can remember my alphabetic letters correctly.

**Comment by PoorPocketsMcNewHold** [comment]
> Thanks for your contribution! 
> Please remove that image from the PR, It could be discussed in another place.

I suppose everything is fine, There still the image from that PR that i can´t seems to be able to delete by using the delete file option from it. Despite i can do that particular thing with the other data.json file.
Capture d'écran Deepin_zone de sélection _20191001111227
If it isn´t just my poor knowledge of Github, I can repull that request.

**Comment by aslafy-z** [comment]
@PoorPocketsMcNewHold I did it for you. To restore a file you can just copy back the file from upstream branch to yours.

**Pull request merged**